### ISSUE TITLE
[QAIT] At UI level Magento2.x (Backend-Admin/Frontend-Customer site) throws Fatal error: Uncaught RuntimeException after  Command- "php magento setup:upgrade" is run

### ISSUE THREAD

**Issue by ramanjeetaulakh-qait** [oxipay/oxipay-magento-2.x#29]
Steps:
1) Installed magento 2.x
2) Created code folder under var/www/html/magento/app
3) Copied oxipay plug in zip under var/www/html/magento/app/code and Unzipped
4) Deleted zipped file
5) Run command: php magento setup:upgrade under var/www/html/magento/bin folder
6) Launch magento2.x Admin UI on browser 
7) Launch magento2.x customer site UI on browser.

Result:
At UI level (Admin/Customer site) Getting Fatal error: Uncaught RuntimeException after  Command- "php magento setup:upgrade" is run

Expected:
There should be no error and UI should launch without any issue.

Screen Cast: http://screencast-o-matic.com/watch/cbhj1LXT07

Document: http://docs.oxipay.com.au/platforms/magento_2/

Observations:
8) Run command: php magento setup:di:compile under /var/www/html/magento/bin
after step 5
9) Launch magento2.x Admin UI on browser 
10) Launch magento2.x customer site UI on browser.

Result:
Able to launch Admin and customer site on browser.

Screen Cast: http://screencast-o-matic.com/watch/cbhj1BXT0t

Tech-Stack:
Ubuntu 16.04.2
PHP 7.0.2
Mysql Ver 14.14 Distrib 5.7.18
Apache/2.4.18

Magento2.1.5
oxipay-magento-2.x_v0.0.14

**Comment by esteban-terem** [comment]
Hi. The root error presented on the screen is *"Can't create directory /var/.../magento/var.../"* which is a permissions error. I think this is caused because you are running the `magento` commands as `root`. Please run all commands as the `magento`/`nginx`/`apache`/`www-data`/... user (depending on the installation).

**Comment by blueninja2** [comment]
Sam/Tao to follow up as a low priority with QA Info Tech to see if the above advise works.

**Comment by ramanjeetaulakh-qait** [comment]
Verified this issue as per the Instructions provided by Esteban above and was able to install oxipay_plugin and was visible at Admin side.

Command: php magento setup:di:compile under /var/www/html/magento/bin
Was not required anymore. 

Marking this bug as closed.
